packages/client/src/App.tsx

```diff
@@ -77,7 +79,11 @@ const Layout = () => {
         <Blurb/>
         <TallStyleContainer>
           <ScrollHook pathEnum='address' pageStart>
-            <AddressForm/>
+            <FieldsContainer.Provider>
+              <AddressFormContainer.Provider>
+                <AddressForm/>
+              </AddressFormContainer.Provider>
+            </FieldsContainer.Provider>
```

Comment:
I don't think you need `FieldsContainer.Provider` here.

Reply:
yeah I know I am not done with the code reviews here yet hehe